### ISSUE TITLE
Build hangs up at running state after upgrading to 8.16.6

### ISSUE THREAD

**Issue by edbergavera** [sameersbn/docker-gitlab#1101]
**OS: Ubuntu 14.04.5**

**Before Upgrade:**

GitLab version: sameersbn/gitlab:8.14.1

GitLab runner: 1.6.1 (installed via apt-get)

Linked containers:

- sameersbn/postgresql:9.4-18

- image: sameersbn/redis:latest

> Notes: All are working fine. No issues.

**After Upgrade:**

GitLab version: sameersbn/gitlab:**8.16.6**

GitLab runner: 1.6.1 (didn't upgrade)
Linked containers (didn't upgrade):

- sameersbn/postgresql:9.4-18

- image: sameersbn/redis:latest

> Notes:
> Builds hangs at **running** state. 
> The `/var/log/syslog` indicates the following error: 
> `Feb 25 22:13:05 ip-172-31-36-1 gitlab-runner[29963]: #033[33mWARN#033[0m[0099] Checking for builds... failed                 #033[33mrunner#033[0m=d8b2957f #033[33mstatus#033[0m=500 Internal Server Error#012<nil>`

**Comment by solidnerd** [comment]
Hey, 
I would recommend to upgrade to runner to the latest version. I guess it's `1.11.0` another tipp is to run a separate container for the runner instead of installing it directly in GitLab.  And you should upgrade your database to the latest version in terms of security and usability because I'm not sure if these version of GitLab works this old Postgres database.

**Comment by edbergavera** [comment]
Hi @solidnerd,

Thanks for your reply. I used the docker compose file when I upgraded GitLab and also generated new secret keys. I came across one online post where it describes almost the same issue and they seem to have resolved it by updating `gitlab-secrets.json` file. And so I tried re-running the docker compose by reverting back the original secret keys below. After this change, the builds or the CI runner is working again 💯 . I don't want to jump into conclusion if the newly generated keys has caused it.

 ```
- GITLAB_SECRETS_DB_KEY_BASE=long-and-random-alphanumeric-string
 - GITLAB_SECRETS_SECRET_KEY_BASE=long-and-random-alphanumeric-string
 - GITLAB_SECRETS_OTP_KEY_BASE=long-and-random-alphanumeric-string
```

I hope this helps.
